packages/workflow/src/Extensions/DateExtensions.ts

```diff
@@ -161,7 +162,11 @@ function isWeekend(date: Date): boolean {
 	return [DAYS.saturday, DAYS.sunday].includes(DateTime.fromJSDate(date).weekday);
 }
 
-function minus(date: Date | DateTime, extraArgs: unknown[]): Date {
+function minus(date: Date | DateTime, extraArgs: unknown[]): Date | DateTime {
+	if (isDateTime(date) && extraArgs.length === 1) {
+		return date.plus(extraArgs[0] as DurationLike);
```

Comment:
shouldn't that be 'minus'? 🤔

Reply:
You are correct. I guess that's what I get for trying to quickly push out a fix on a Monday morning with no coffee :facepalm: